### PR TITLE
[PREVIEW] Feature/sl 1969/duration and sessions hearing details

### PR DESCRIPTION
**Pull request by andrewtaylorkainos**
### Change description ###
Changed Schema to include 'number_of_sessions' and 'is_multisession'.
Added new fields to create/read/update requests.
Migration script included.
Tests amended to include new fields.

**Does this PR introduce a breaking change?** (check one with "x")

```
[X] Yes - New Schema
[ ] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/sandl/snlevents/model/db/Hearing.java [hunk 1]
```diff
@@ -65,6 +65,11 @@
 
     private Duration duration;
 
+    private Integer numberOfSessions;
+
+    @Column(name = "is_multisession")
```
Comment (leszekgu) [inline src/main/java/uk/gov/hmcts/reform/sandl/snlevents/model/db/Hearing.java]: is it only me or another _ would be better: is_multi_session ?

Comment (andrewtaylorkainos) [reply src/main/java/uk/gov/hmcts/reform/sandl/snlevents/model/db/Hearing.java]: technically the English is 'multi-session' and not 'multi session', which the _ would suggest :D

File: src/main/java/uk/gov/hmcts/reform/sandl/snlevents/model/request/CreateHearingRequest.java [hunk 2]
```diff
@@ -46,6 +46,9 @@
     @Min(1)
     private Integer numberOfSessions;
 
+    @NotNull
+    private Boolean isMultiSession;
```
Comment (r-kruk-kainos) [inline src/main/java/uk/gov/hmcts/reform/sandl/snlevents/model/request/CreateHearingRequest.java]: Why not just primitive type?

File: src/main/java/uk/gov/hmcts/reform/sandl/snlevents/model/request/UpdateListingRequest.java [hunk 3]
```diff
@@ -39,6 +40,13 @@
     @MinDuration(minMinutes = 1)
     private Duration duration;
 
+    @NotNull
+    @Min(1)
+    private Integer numberOfSessions;
```
Comment (r-kruk-kainos) [inline src/main/java/uk/gov/hmcts/reform/sandl/snlevents/model/request/UpdateListingRequest.java]: Same as above and in some other places too.

Comment (andrewtaylorkainos) [reply src/main/java/uk/gov/hmcts/reform/sandl/snlevents/model/request/UpdateListingRequest.java]: Done

File: src/main/java/uk/gov/hmcts/reform/sandl/snlevents/model/request/CreateHearingRequest.java (hunk 2 above)
Comment (andrewtaylorkainos) [reply src/main/java/uk/gov/hmcts/reform/sandl/snlevents/model/request/CreateHearingRequest.java]: Done

**Comment by andrewtaylorkainos** [comment]
In the primitive type, the 'is' gets ignored in the variable name and the JSON response automatically becomes 'multiSession'.
Because of this, i renamed all of the 'isMultiSession' variables to 'multiSession' so that they match the JSON response

**Pull request merged**